ch15/meadowlark.js

```diff
@@ -241,7 +241,13 @@ require('./routes.js')(app);
 
 var Attraction = require('./models/attraction.js');
 
-var rest = require('connect-rest');
+// API configuration
+var apiOptions = {
+    context: ''
+};
+
+var Rest = require('connect-rest');
+let rest = Rest.create( apiOptions )
```

Comment:
Why `let` here and `var` above?  I'm all about `const` these days, but this book was written before it was easy to do all of that without transcompilation.  This will all be pretty significantly different in the 2nd edition.  Also, I'm not fond of unnecessary identifiers...other than calling `create`, you don't use `Rest` anywhere else.  So I would write this as:

    const rest = require('connect-rest').create({ context: '' })